snippets/graphql/create-component/README.md

```diff
@@ -0,0 +1,45 @@
+# How to use this
+
+This query will retrieve a list of metric definitions for your site.
+
+Replace `cloudId` below in the variables section with the cloudId for your site and execute the query. You can use [the GraphQL explorer](https://developer.atlassian.com/cloud/compass/graphql/explorer/) to run this query and explore [the Compass API](https://developer.atlassian.com/cloud/compass/graphql/) further.
```

Comment:
Maybe mention the "type" here as well and link to https://developer.atlassian.com/cloud/compass/components/what-is-a-component/#component-types